config/locales/en_etm.yml

```diff
@@ -336,12 +336,12 @@ en:
       manage: 'Manage access'
       manage_description:
         Here you can manage who has access to your scenario and in which manner.
-        Click the 'Invite' button to give someone access to your scenario,
+        Click the 'Invite users' button to give someone access to your scenario,
```

Comment:
Maybe 'invite others'? My fear is that with saying 'user's, people may think they can only invite registered users. But they can also invite people that are new to the ETM

Reply:
I agree, 'others' would be better